src/03-error-handling-async/index.test.ts

```diff
@@ -1,30 +1,42 @@
 // Uncomment the code below and write your tests
-// import { throwError, throwCustomError, resolveValue, MyAwesomeError, rejectCustomError } from './index';
+import {
+  throwError,
+  throwCustomError,
+  resolveValue,
+  MyAwesomeError,
+  rejectCustomError,
+} from './index';
 
 describe('resolveValue', () => {
   test('should resolve provided value', async () => {
-    // Write your test here
+    expect.assertions(1);
+    const data = 'Some really good data';
+    await expect(resolveValue(data)).resolves.toEqual(data);
   });
 });
 
 describe('throwError', () => {
   test('should throw error with provided message', () => {
-    // Write your test here
+    const t = () => throwError('Error message');
+    expect(t).toThrow(new Error('Error message'));
   });
 
   test('should throw error with default message if message is not provided', () => {
-    // Write your test here
+    const t = () => throwError();
+    expect(t).toThrow(new Error('Oops!'));
   });
 });
 
 describe('throwCustomError', () => {
   test('should throw custom error', () => {
-    // Write your test here
+    const t = () => throwCustomError();
+    expect(t).toThrow(new MyAwesomeError());
```

Comment:
The point is to verify that error thrown is an instance of `MyAwesomeError`.
Current implementation will accept any error with `message`: `This is my awesome custom error!` 

Required changes:
```suggestion
    expect(t).toThrow(MyAwesomeError);
```